test/unit/loggingManagerTests.js

```diff
@@ -336,10 +383,10 @@ describe('LoggingManager', function () {
 
     describe('when time value returned that is more than current time', function () {
       describe('when level is already set', function () {
-        beforeEach(function () {
+        beforeEach(async function () {
           this.bunyanLogger.level.returns(10)
-          this.Fs.readFile.yields(null, (this.start + 1000).toString())
-          this.logger.checkLogLevel()
+          this.Fs.promises.readFile.returns((this.start + 1000).toString())
+          await this.logger.checkLogLevelFile()
```

Comment:
There is a mixture of `async` setups and regular synchronous functions. Both seem to be working fine, can we agree on one style nevertheless?